R/distribution_formulas.R

```diff
@@ -0,0 +1,356 @@
+#' Validaity checks for new_data_distribution and new_process_distribution
+#' 
+check_distribution_validity <- function(args){
+  list2env(args, envir = environment())
```

Comment:
Just an FYI this will lead to warning checks on CRAN, not that we are trying for CRAN right now, because there is no way to know what is present in "args".

Reply:
good catch! I like to try to make the code as CRAN ready as possible so we have less to fix later if we ever decide to submit.

Reply:
removed use of list2env